test/message.test.js

```diff
@@ -43,17 +44,17 @@ describe('message', () => {
   });
 
   it('avatar should be set with provided user', () => {
-    message.user = user;
+    setUser(message);
```

Comment:
Please consider rewriting this test into several separate tests. Each test should ideally check one property.

Example:
```js
it('should propagate user name to the avatar name', () => {
  message.userName = 'Joan Doe';
  expect(avatar. name).to.be.equal('Joan Doe');
});
```

Then we would be able to also remove the `setUser` method which does could make code less easy to read especially in case when the test suite grows over time.

By having separate tests for different properties, testing one property at at time and keeping the assignment close to the assertion, we make code more readable.

Also, it is recommended to check for properties on the element, and not attributes.

Reply:
Done. It is now separated, and the `setUser(message)` function is removed. Properties are tested instead of attribute.